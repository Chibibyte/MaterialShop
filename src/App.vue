<template>
  <v-app>
    <div class="secondary fill-height">
      <router-view />
    </div>
  </v-app>
</template>

<script>
export default {
  name: "App"
};
</script>
