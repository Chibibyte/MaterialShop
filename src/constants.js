export default {
    BUTTONICONS: {
        AUTHENTICATION: 'mdi-login-variant',
        ADDRESS: 'mdi-package-variant',
        BANKACCOUNT: 'mdi-bank',
        SHIPPING: 'mdi-truck',
        PAYMENT: 'mdi-cash',
        SUMMARY: 'mdi-text-box-check-outline',
        CHECKOUT_COMPLETE: 'mdi-check',
        ADD: 'mdi-plus',
        REMOVE: 'mdi-minus',
        CANCEL: 'mdi-window-close',
        CONFIRM: 'mdi-check',
        SHOPPING: 'mdi-shopping',
        SUCCESS: 'mdi-check',
        FAILURE: 'mdi-window-close',
        WARNING: 'mdi-alert-circle-outline'
    }
}