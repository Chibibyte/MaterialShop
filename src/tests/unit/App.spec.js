
import Vuex from 'vuex'
import { shallowMount, createLocalVue } from '@vue/test-utils';
import App from '@/App.vue';
import constants from '@/constants.js'

const localVue = createLocalVue();
localVue.use(Vuex);
localVue.prototype.$constants = constants;

let stubs = [
    'v-app',
    'router-view'
]
let wrapper;

beforeEach(() => {
    wrapper = shallowMount(App, { stubs, localVue });
})

// initial
describe('Component', () => {
    test('is instantiated', () => {
        expect(wrapper.isVueInstance).toBeTruthy();
    });

    test('default snapshot', () => {
        expect(wrapper.element).toMatchSnapshot();
    })
});




