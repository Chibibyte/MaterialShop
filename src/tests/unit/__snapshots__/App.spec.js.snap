// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component default snapshot 1`] = `
<v-app-stub>
  <div
    class="secondary fill-height"
  >
    <router-view-stub />
  </div>
</v-app-stub>
`;
